caffe2/distributed/redis_store_handler.h

```diff
@@ -25,6 +25,8 @@ class CAFFE2_API RedisStoreHandler : public StoreHandler {
 
   virtual int64_t getNumKeys() override;
 
+  virtual bool deleteKey(const std::string& key) override;
```

Comment:
Is this part of API still used by any active code path?

Reply:
It seems like it is used in some examples/tests in the Caffe2 codebase as well as in one CV application internally, but it's not used in PyTorch code.

Reply:
I see.